Linux_scripts/RHUI_repo_validation_scripts/repo_check.py

```diff
@@ -75,10 +88,10 @@ def yum_clean_up():
   
```

Comment:
except block is missing, add it and capture error in case it caches one

Reply:
Added except block wherever missing